app/models/spree/generate_pos_order.rb

```diff
@@ -56,11 +56,27 @@ def add_line_items(order, pos_order)
 
           order.line_items << line_item
           line_item.order = order
+          line_item.adjustments = build_adjustments(item, line_item, order)
         end
       end
       order.save!
     end
 
+    def build_adjustments(shopify_line_item, spree_line_item, order)
+      adjustments = []
+      shopify_line_item.tax_lines.each do |tax|
+        adjustment = Spree::Adjustment.new
+        adjustment.amount = tax.price
+        adjustment.label = tax.title
+        adjustment.order = order
+        adjustment.included = false
+        adjustment.adjustable = spree_line_item
```

Comment:
Could this be reduced in complexity via ActiveRecord association methods? For example: `spree_line_item.adjustments.tax.create`?